integration-cli/docker_cli_build_test.go

```diff
@@ -420,12 +411,12 @@ func (s *DockerSuite) TestBuildEnvEscapes(c *check.C) {
     `,
 		true)
 
-	out, _, err := runCommandWithOutput(exec.Command(dockerBinary, "run", "-t", name))
-
 	if err != nil {
```

Comment:
Oh actually, no check was done on the ``err`` out of the ``buildImage`` just a little *up*, that's probably why I didn't remove the check on err.